CodeGeneration/Sources/Utils/CodeGenerationFormat.swift

```diff
@@ -66,7 +66,7 @@ public class CodeGenerationFormat: BasicFormat {
 
   public override func visit(_ node: MemberDeclListItemSyntax) -> MemberDeclListItemSyntax {
     let formatted = super.visit(node)
-    if node.indexInParent != 0 && !node.decl.is(EnumCaseDeclSyntax.self) {
+    if node.index != node.parent?.children(viewMode: .sourceAccurate).first?.index && !node.decl.is(EnumCaseDeclSyntax.self) {
```

Comment:
I don’t actually think you need to compare the indices here, comparing the nodes themselves should be sufficient

```suggestion
    if node != node.parent?.children(viewMode: .sourceAccurate).first? && !node.decl.is(EnumCaseDeclSyntax.self) {
```